pkg/scheduler/internal/cache/cache.go

```diff
@@ -750,7 +750,7 @@ func (cache *cacheImpl) cleanupAssumedPods(now time.Time) {
 				"pod", klog.KObj(ps.pod))
 			continue
 		}
-		if now.After(*ps.deadline) {
+		if (cache.ttl != 0) && (now.After(*ps.deadline)) {
```

Comment:
```suggestion
		if cache.ttl != 0 && now.After(*ps.deadline) {
```

Reply:
this still looks like the old code

Reply:
@alculquicondor Please review, I have incorporate your comment.